core/src/test/java/org/elasticsearch/cluster/shards/ClusterSearchShardsIT.java

```diff
@@ -154,4 +157,14 @@ public void testClusterSearchShardsWithBlocks() {
             disableIndexBlock("test-blocks", SETTING_BLOCKS_METADATA);
         }
     }
+
+    public void testClusterSearchShardsWithAliases() {
```

Comment:
we can get rid of it once we have the unit test for serialization. Going to push that one.